test/cleanup.go

```diff
@@ -67,6 +67,12 @@ func TearDown(clients *Clients, names *ResourceNames) {
 			[]string{names.Config},
 			[]string{names.Service},
 		)
+		clients.ServingClient.WaitForDeletion(
+			[]string{names.Route},
+			[]string{names.Config},
```

Comment:
Are we certain this actually blocks until all pods are removed? That should take like 50s per test roughly.

Reply:
It blocks on services, configs, and routes, I believe the configs are what block on pods.

Reply:
oh, hah, sorry my bad, dont know how I missed what pkg I was in there!